src/test/osd/osd-scrub-repair.sh

```diff
@@ -908,7 +908,12 @@ function TEST_corrupt_scrub_replicated() {
 EOF
 
     jq "$jqfilter" $dir/json | python -c "$sortkeys" | sed -e "$sedfilter" > $dir/csjson
-    diff -y $termwidth $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
+    if [ `uname` != FreeBSD ]; then
+        diff -y $termwidth $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
```

Comment:
i'd suggest always use plain `diff` without `-y` option or just `cmp` to compare `checkcsjson` and `csjson`. and could also wrap this logic with a simple helper function, as it is repeating itself multiple times in this shell script.

@dzafman what do you think?